src/config/options.js

```diff
@@ -101,6 +101,61 @@ const rootSection = section(
                 "": { files: [] }, // Use `all` set with default values if sets were not specified in a config
             },
         ),
+
+        devServer: section({
+            command: options.optionalString("devServer.command"),
+            env: options.optionalObject("devServer.env"),
+            args: options.optionalArray("devServer.args"),
+            cwd: options.optionalString("devServer.cwd"),
+            logs: options.optionalBoolean("devServer.logs"),
+            readinessProbe: option({
+                defaultValue: defaults.devServer.readinessProbe,
+                validate: value => {
+                    if (typeof value === "function" || value === null) {
+                        return;
+                    }
+
+                    if (!_.isPlainObject(value)) {
+                        throw new Error("devServer.readinessProbe must be a function, object or null");
```

Comment:
let's wrap the variable in quotation marks as we do above in this file.

`devServer.readinessProbe` -> `"devServer.readinessProbe"`